drivers/bmp180/bmp180_saul.c

```diff
@@ -39,9 +39,9 @@ static int read_pressure(void *dev, phydat_t *res)
 {
     bmp180_t *d = (bmp180_t *)dev;
 
-    res->val[0] = bmp180_read_pressure(d) / 10;
+    res->val[0] = bmp180_read_pressure(d) / 100;
```

Comment:
you did the same for bmx280, see my comment above. Btw. SAUL always introduces some kind of compromise on resolution and accuracy, if one cares about that: don't use SAUL?